pkg/apigw/helpers.go

```diff
@@ -21,3 +25,35 @@ func helperDefaultResponse(opt *options.ApigwOpt) http.HandlerFunc {
 		}
 	}
 }
+
+func helperMethodNotAllowed(opt *options.ApigwOpt, pr *profiler.Profiler) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		addToProfiler(opt, pr, r, http.StatusMethodNotAllowed)
+
+		if opt.LogEnabled {
+			// Say something friendly when logging is enabled
+			http.Error(w, devHelperResponseBody, http.StatusTeapot)
```

Comment:
This is nice :)
But why not properly returning 405 even when logging? The fact that logging is enabled does not affect (other) behaviour of the handler and its responses , correct?

Reply:
I just wanted a teapot, I agree with you though